hack/deploy-example-secret.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
```

Comment:
Recommendation to use `set -eou pipefail` by default. This helps a lot with pipes and errors among other things.